packages/dds/tree/src/shared-tree/schematizingTreeView.ts

```diff
@@ -409,7 +409,7 @@ export class SchematizingSimpleTreeView<
  * @remarks Currently, all contexts are also {@link SchematizingSimpleTreeView}s.
  * Other checkout implementations (e.g. not associated with a view) may be supported in the future.
  */
-function getCheckout(context: TreeBranch): TreeCheckout {
+export function getCheckout(context: TreeBranch): TreeCheckout {
```

Comment:
*shrug* Honestly, if/when we add view-less branches, there's going to be enough that will go wrong that this will just be the tip of the iceberg. So yea, it will save a little time, but it's not particularly important. As it currently stands, the function is being exported from `schematizingTreeView`, so anyone wishing to use this to "avoid depending on SchematizingTreeView" will be... depending on schematizingTreeView.ts to import the function. Lol.